packages/cozy-stack-client/src/OAuthClient.js

```diff
@@ -364,6 +364,19 @@ class OAuthClient extends CozyStackClient {
     return new AccessToken(result)
   }
 
+  /**
+   * Fetches a new session code. Only usable by the Flagship application
+   *
+   * @throws {NotRegisteredException} When the client isn't certified to be the Flagship application
+   * @returns {Promise} A promise that resolves with a new session_code
```

Comment:
can you document the return please. I don't care that it's a promise, I care about the structure of the return ;)

Reply:
I'm not very used to JSdoc syntax, what do you think of the update?